libdrgn/configure.ac

```diff
@@ -37,6 +37,21 @@ AS_CASE(["x$enable_openmp"],
 AC_SUBST(OPENMP_CFLAGS)
 AC_SUBST(OPENMP_LIBS)
 
+AC_ARG_ENABLE([asan],
+	      [AS_HELP_STRING([--enable-asan@<:@=ARG@:>@],
+			      [use ASAN. ARG may be yes or no.
+			       @<:@default=no@:>@])],
```

Comment:
Since this is a simple yes/no option, we can just do `AS_HELP_STRING([--enable-asan], [enable AddressSanitizer])`.